cirq/google/devices/known_devices.py

```diff
@@ -61,55 +65,77 @@ def create_device_proto_from_diagram(
         gate_sets: Optional[Iterable[
             serializable_gate_set.SerializableGateSet]] = None,
         durations_picos: Dict[str, int] = None,
+        out: Optional[device_pb2.DeviceSpecification] = None,
 ) -> device_pb2.DeviceSpecification:
-    """
-    Parse ASCIIart device layout into DeviceSpecification proto containing
-    information about qubits and targets.  Note that this does not populate
-    the gate set information of the proto.  This function also assumes that
-    only adjacent qubits can be targets.
+    """Parse ASCIIart device layout into DeviceSpecification proto.
 
-    Args:
-            ascii_grid: An ASCII version of the grid, see _parse_device for
-                    details
-            gate_set: A serializable gate_set object that is used to define
-                    the translation between gate ids and cirq Gate objects
-            durations_picos: A dictionary with keys of gate ids and values of
-                    the duration of that gate in picoseconds
+    This function also assumes that all pairs of adjacent qubits are valid
```

Comment:
nit s/also//

(no other assumption is given in preceding text, so it's unclear what "also" refers to)

Reply:
Fixed.